control-plane-operator/controllers/hostedcontrolplane/manifests/kubevirt.go

```diff
@@ -38,6 +38,15 @@ func KubevirtCSIDriverTenantKubeConfig(ns string) *corev1.Secret {
 	}
 }
 
+func KubevirtCSIDriverInfraKubeConfig(ns string) *corev1.Secret {
+	return &corev1.Secret{
+		ObjectMeta: metav1.ObjectMeta{
+			Name:      "kubevirt-csi-controller-infra-kubeconfig",
+			Namespace: ns,
+		},
+	}
+}
+
```

Comment:
is this used?

Reply:
thanks, removed.
initially i thought i need to create the secret, same as the `kubevirt-csi-controller-tenant-kubeconfig` secret that is created by CPO. but we can leverage the existing `kubevirt-infra-credentials` in our case.